includes/content-distribution/class-distributed-post.php

```diff
@@ -0,0 +1,203 @@
+<?php
+/**
+ * Newspack Network Distributed Post.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack_Network\Content_Distribution;
+
+use Newspack_Network\Content_Distribution;
+use WP_Post;
+use WP_Error;
+
+/**
+ * Distributed Post Class.
+ */
+class Distributed_Post {
+	/**
+	 * The post meta key for the distributed post configuration.
+	 */
+	const DISTRIBUTED_POST_META = 'newspack_network_distributed';
+
+	/**
+	 * The post ID.
+	 *
+	 * @var int
+	 */
+	public $ID = 0;
```

Comment:
It's a public var, whereas `$post` is protected. To avoid the duplication I changed the strategy to deprecate this var and add a public `get_post()` in f755a01ccc65854929b8402913e149f49b1853e1. WDYT?

Reply:
I like it! 👍